tests/email/email_parsing_test.php

```diff
@@ -73,7 +73,8 @@ public function setUp()
 		$phpbb_container->set('ext.manager', $extension_manager, phpbb_mock_container_builder::SCOPE_PROTOTYPE);
 
 		$context = new \phpbb\template\context();
-		$phpbb_container->set('template.twig.extensions.collection', array(new \phpbb\template\twig\extension($context, $user)), phpbb_mock_container_builder::SCOPE_PROTOTYPE);
+		$twig_extension = new \phpbb\template\twig\extension($context, $user);
```

Comment:
Actually, this part is wrong here (as well as in the previous PR) as the 2nd parameter should be ```$lang``` for 3.2.